lib/command.js

```diff
@@ -108,7 +108,26 @@ class Command {
      * @private
      */
     static _run(commandName, argv, extensions) {
+        debug('running command prep');
+
+        try {
+            const path = require('path');
+            const dir = path.resolve(argv.dir);
+
+            if (this.ensureDir) {
+                fs.ensureDirSync(dir);
+            }
+
+            process.chdir(dir);
+        } catch (error) {
+            const chalk = require('chalk');
+            console.warn(chalk.yellow(`Error using "${argv.dir}" (${error.code || error.message || error}) - staying in"${process.cwd()}"`));
```

Comment:
couple things about this:

1) would it be possible to move this down further in the run? that way this could have access to the UI class instead of needing to do console.warn
2) Not 100% sure about just "staying in the current directory" - that feels like it would be unexpected behavior to me. If I run a command `ghost <cmd> --dir <some dir>` then I was intentionally wanting to run it in a specific directory, so having Ghost decide to stay in the current directory rather than error & exit seems surprising.

Reply:
1. Definitely!
2. Makes sense, will do